src/attestation/Attestation.ts

```diff
@@ -89,6 +62,32 @@ export default class Attestation implements IAttestation {
     return revoke(claimHash, identity)
   }
 
+  /**
+   * Creates a new instance of this Attestation class from the given interface.
+   */
+  public static fromAttestationInterface(obj: IAttestation): Attestation {
+    return new Attestation(
+      obj.claimHash,
+      obj.cTypeHash,
+      obj.owner,
+      obj.delegationId,
+      obj.revoked
+    )
+  }
+
+  public static fromRequest(
```

Comment:
`fromRequestAndIdentity`. Also it is enough to expect the public identity interface

Reply:
'fromRequestAndPublicIdentity' would be more fitting?